changes.txt

```diff
@@ -46,6 +46,7 @@ Released: N/A
 * Added love.system.getPreferredLocales.
 * Added love.localechanged callback.
 * Added World:getFixturesInArea().
+* Added CMake build ability.
```

Comment:
Windows CMake has always worked and continues to work. I don't think we really want to officially support multiple build paths on each platform (although unofficial support is fine), so I'd rather not have this in the changelog.